src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -215,6 +215,12 @@ public void testViewPhotoAndAjaxForLargeScaledSession() throws Exception {
        
         resultsPage.verifyHtmlAjax("/instructorFeedbackResultsAjaxByQuestion.html");
         
+        ______TS("test view photo for view by questions");
+        
+        resultsPage.removeNavBar();
+        resultsPage.hoverClickAndViewPhotoOnTableCell(0, 0, 0, "studentProfilePic?studentemail={*}&courseid={*}&user=CFResultsUiT.instr");
+        resultsPage.hoverClickAndViewPhotoOnTableCell(0, 0, 2, "profile_picture_default.png");
```

Comment:
Once you are done with this, just make sure dev is green and I will merge this one in :)